src/main/java/seedu/contax/logic/commands/RangeCommand.java

```diff
@@ -34,6 +37,8 @@
             + PREFIX_PHONE + "12345678 "
             + PREFIX_ADDRESS + "new address ";
 
+    private final Logger logger = LogsCenter.getLogger(LogicManager.class);
```

Comment:
What is the purpose of this logging? You are introducing a cyclic dependency and I do not see any other command classes doing it.

Reply:
The original logger seems only record entire command, logging the individual new command may help in debuging or something

Reply:
Ohh the cyclic dependency I change the class later